packages/website/src/components/breadcrumbs.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import titleCase from 'title-case';
+import Breadcrumbs, { BreadcrumbsItem } from '@atlaskit/breadcrumbs';
+import styled from '@emotion/styled';
+import { gridSize, math } from '@atlaskit/theme';
+
+const Header = styled.div`
+  left: 0;
+  right: 0;
+  padding: ${math.multiply(gridSize, 4)}px ${math.multiply(gridSize, 5)}px;
+  padding-bottom: ${math.multiply(gridSize, 1.5)}px;
+`;
+
+type BreadcrumbsNavProps = {
+  pagePath: string;
+};
+
+const BreadcrumbsNav = ({ pagePath }: BreadcrumbsNavProps) => {
+  const pgPath = pagePath.replace('/index.js', '').replace('.js', '');
```

Comment:
Where do we have pagePaths that include `.js` - that seems like a bug elsewhere in the system.

Reply:
Similar, are all the paths absolute, or do we get relative paths? I assume this only works if we get absolute paths.

Reply:
this is the absolute file path. The file extension gets replaced in `getImportPath` when converting to a relative path, but I'm grabbing `pagePath` before that happens so I am doing the replacement in the breadcrumbs component

Reply:
Hmkay. Does this imply we should lift that conversion higher up?

Reply:
Possibly, but I'm not sure if that'll break anything